config/environments/production.rb

```diff
@@ -24,7 +24,8 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
+  # config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
What's this about?

Reply:
Some sort of JS transpiler issue without which deployment fails in managed environments (I thought that I could get away without it after I got past the `yarn` hurdle, but tested it this morning and failed, so it has to stay):
<img width="738" alt="Screenshot 2021-12-17 at 13 11 12" src="https://user-images.githubusercontent.com/53006/146542994-fdb5432c-74fd-4107-af87-78a12db35ec7.png">

```
Uglifier::Error: Unexpected token: name (LocalAutoSave). To use ES6 syntax, harmony mode must be enabled with Uglifier.new(:harmony => true).
```

Reply:
TIL, we change this in Pro 3 years go. So this change brings back parity.